src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -31,6 +31,7 @@
 [assembly: InternalsVisibleTo("OrleansHost")]
 [assembly: InternalsVisibleTo("OrleansAzureUtils")]
 [assembly: InternalsVisibleTo("OrleansAWSUtils")]
+[assembly: InternalsVisibleTo("OrleansGoogleUtils")]
```

Comment:
The `ErrorCode` is one case. There are others. As you can see, AWS and Azure utils do the same since they rely on it.